### ISSUE TITLE
dismissOnTimeout not working for me

### ISSUE THREAD

**Issue by jparrilla** [insites/cookieconsent#323]
I am using a really basic version of the JS as seen here:

```javascript
window.addEventListener("load", function(){
    window.cookieconsent.initialise({
        "palette": {
            "popup": {
                "background": "#000"
            },
            "button": {
                "background": "#f1d600"
            }
        },
        "position": "bottom-right",
        "dismissOnTimeout": 10000
    })
});
```

The dismissOnTimeout does not seem to be doing anything at all. I looked through the source and it seems as though it should work as long is typeof returns 'number' and its >= 0. The usage can be seen here in the built in applyAutoDismiss function:

```javascript
var delay = this.options.dismissOnTimeout;
if (typeof delay == 'number' && delay >= 0) {
    this.dismissTimeout = window.setTimeout(function() {
        setStatus(cc.status.dismiss);
    }, Math.floor(delay));
}
```

Anyone else experience this?

**Comment by Apollinaire** [comment]
In my case the `dismissOnTimeOut` setting sets the cookie to `dismiss` but the popup does not disappear. I don't know if that is intended or not.

**Comment by brankoiliccc** [comment]
@Apollinaire  same here. Anyone found a solution to how to close popup when dismissOnTimeOut time passes and consent is given?

**Comment by Apollinaire** [comment]
I did not, and now I think it's more fair to not close the popup automatically after a given time. The popup will stop appearing at the next page load anyway so I think it's already enough "choosing for the user" like that. 
The base concept of RGPD is to be more transparent and give more choice to the users about their personal data and cookie usage

**Comment by Dolondro** [comment]
This should be fixed as of 3.1.0
